README.md

```diff
@@ -153,15 +153,19 @@ dcos-docker create /tmp/dcos_generate_config.sh \
     --cluster-id default
 ```
 
-2. Run `pytest` on a master node.
+2. Run `dcos-docker wait`
+
+Running wait command allows to make sure that the cluster is set up properly before any other actions that could otherwise cause errors, such as, for example, pytest not finding required test files.
```

Comment:
We could re-use `dcos-e2e` document.

```
Wait for DC/OS to start.
Running wait command allows to make sure that the cluster is set up properly before any other actions that could otherwise cause errors in `pytest` command in the next step.
```